brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
(Other side note, `has_cart()`, which you were probably basing this on, clearly doesn't follow that convention either :-p and has the same problem. Maybe that could be fixed as part of this too?)